msal/build/Sdl7.0.ruleset

```diff
@@ -1,143 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="SDL 7.0" Description="SDL 7.0-prescribed required and recommended checks." ToolsVersion="11.0">
-  <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
```

Comment:
>ManagedCodeAnalysis [](start = 41, length = 19)

This ruleset was used only by a test project. Note that SDL rules do run as part of the build, but through the old FxCop (i.e. binary level analysis, which, arguably, is more powerful)